package.json

```diff
@@ -25,75 +25,76 @@
     "stream": "./node_modules/stream-browserify"
   },
   "dependencies": {
-    "@angular/animations": "^18.2.8",
-    "@angular/cdk": "^18.2.9",
-    "@angular/common": "^18.2.8",
-    "@angular/compiler": "^18.2.8",
-    "@angular/core": "^18.2.8",
-    "@angular/fire": "18.0.0",
-    "@angular/forms": "^18.2.8",
-    "@angular/localize": "^18.2.8",
-    "@angular/material": "^18.2.9",
-    "@angular/material-moment-adapter": "^18.2.9",
-    "@angular/platform-browser": "^18.2.8",
-    "@angular/platform-browser-dynamic": "^18.2.8",
-    "@angular/router": "^18.2.8",
-    "@angular/service-worker": "^18.2.8",
+    "@angular/animations": "^19.1.1",
+    "@angular/cdk": "^19.1.0",
+    "@angular/common": "^19.1.1",
+    "@angular/compiler": "^19.1.1",
+    "@angular/core": "^19.1.1",
+    "@angular/forms": "^19.1.1",
+    "@angular/fire": "^19.0.0",
+    "firebase": "^10.12.0",
+    "@angular/localize": "^19.1.1",
+    "@angular/material": "^19.1.0",
+    "@angular/material-moment-adapter": "^19.1.0",
+    "@angular/platform-browser": "^19.1.1",
+    "@angular/platform-browser-dynamic": "^19.1.1",
+    "@angular/router": "^19.1.1",
+    "@angular/service-worker": "^19.1.1",
     "@aytek/material-color-picker": "^1.0.4",
-    "@casl/ability": "^6.7.1",
-    "@casl/angular": "^8.2.8",
-    "@faker-js/faker": "^8.4.1",
-    "@fortawesome/angular-fontawesome": "^0.15.0",
-    "@fortawesome/fontawesome-svg-core": "^6.6.0",
-    "@fortawesome/free-regular-svg-icons": "^6.6.0",
-    "@fortawesome/free-solid-svg-icons": "^6.6.0",
+    "@casl/ability": "^6.7.3",
+    "@casl/angular": "^9.0.3",
+    "@faker-js/faker": "^9.4.0",
+    "@fortawesome/angular-fontawesome": "^1.0.0",
+    "@fortawesome/fontawesome-svg-core": "^6.7.2",
+    "@fortawesome/free-regular-svg-icons": "^6.7.2",
+    "@fortawesome/free-solid-svg-icons": "^6.7.2",
     "@ngneat/until-destroy": "^10.0.0",
-    "@sentry/angular": "^8.48.0",
+    "@sentry/angular": "^8.50.0",
```

Comment:
@Abhinegi2 , can you go through the package.json once more (e.g. with `npm outdated`) and upgrade packages that since then have an even newer version released? (sentry is on 8.55.0 now, for example)